tests/integration/suite/actors/http/ttl.go

```diff
@@ -99,16 +102,22 @@ func (l *ttl) Run(t *testing.T, ctx context.Context) {
 
 	req, err := http.NewRequest(http.MethodPost, daprdURL+"/v1.0/actors/myactortype/myactorid/method/foo", nil)
 	require.NoError(t, err)
-	resp, err := client.Do(req)
-	require.NoError(t, err)
-	assert.NoError(t, resp.Body.Close())
-	assert.Equal(t, http.StatusOK, resp.StatusCode)
+
+	assert.EventuallyWithT(t, func(c *assert.CollectT) {
+		resp, err := client.Do(req)
+		require.NoError(c, err)
+		assert.NoError(c, resp.Body.Close())
+		assert.Equal(c, http.StatusOK, resp.StatusCode)
+	}, time.Second*10, time.Millisecond*100, "actor not ready")
+
+	require.False(t, t.Failed())
```

Comment:
I'm pretty sure we can use `require.EventuallyWithT` instead.

Reply:
Changed